### ISSUE TITLE
Improve accessibility

### ISSUE THREAD

**Issue by denisw** [tannerlinsley/react-table#130]
Currently, the default react-table components output just nested `<div>` tags, which means that screen readers and other assistive technologies don't recognize that the markup actually represents a table. Thus, the different parts should have [WAI-ARIA](https://www.w3.org/TR/wai-aria) `role` attributes to expose this information. Relevant here is the [grid](https://www.w3.org/TR/wai-aria/roles#grid) role and its accompanying roles (`row`, `gridcell` etc.).

**Comment by tannerlinsley** [comment]
I would love to have more aria compliance, so anyone who would like to work on this, let me know.
